global.json

```diff
@@ -5,12 +5,11 @@
   "tools": {
     "dotnet": "7.0.100-alpha.1.21558.2",
     "runtimes": {
```

Comment:
@Tratcher could you please have a look at the Server Comparison test failures❔ They probably relate to the missing .dotnet/x86 folder on Windows build agents w/ the changes in this file. But, I'd appreciate your thoughts.

Reply:
Yes, it makes sense these tests would fail without the x86 runtime. I'm a little surprised most of them gracefully fell back to x64 and only failed checking the output. Those are the easiest variations to remove. The one that didn't gracefully fall back is the one we should preserve somehow, since x86 actually mattered in that scenario.
`HelloWorld(variant: Server: IISExpress, TFM: net7.0, Type: Portable, Arch: x86, Host: InProcess)`

Reply:
Let's table this unless we find we _must_ remove the "dotnet/x86" entry at some point in the future.